### ISSUE TITLE
pythorch to onnx conversion , accuracy loss

### ISSUE THREAD

**Issue by Bea07** [junyanz/pytorch-CycleGAN-and-pix2pix#1534]
Hi,
I have tried to convert a CycleGAN model to onnx and I found a relevant decrease of accuracy.
Anyone with some advice, suggestions and if you experienced something similar.
I have tried many tests, different parameters for torch.onnx.export() but no improvements.
Have you come into something similar?
thks
Bea

**Comment by junyanz** [comment]
I am not familiar with onnx conversion. But the default CycleGAN uses instance normalization.

**Comment by Bea07** [comment]
> I am not familiar with onnx conversion. But the default CycleGAN uses instance normalization.

yep indeed the pretrained cycleGAN es horse2zebra don't have accuracy loss ( for pix2pix GAN examples es. sat2map it decreased a lot, tho). I indeed edit a bit the CycleGan model for learning the structure better, and i could not find why there were this output differences.
